django-stubs/template/defaultfilters.pyi

```diff
@@ -2,13 +2,18 @@ from collections.abc import Callable
 from datetime import date as _date
 from datetime import datetime
 from datetime import time as _time
-from typing import Any
+from typing import Any, TypeVar
 
+from django.template.library import Library
 from django.utils.safestring import SafeString
+from typing_extensions import Concatenate, ParamSpec
 
-register: Any
+_P = ParamSpec("_P")
+_R = TypeVar("_R")
 
-def stringfilter(func: Callable) -> Callable: ...
+register: Library
+
+def stringfilter(func: Callable[Concatenate[str, _P], _R]) -> Callable[Concatenate[object, _P], _R]: ...
```

Comment:
How come `object` is in the return type? Is it in place of `Any` or some other reason?

Reply:
Mainly because `stringfilter` applies `str` on the first argument, and `str` expects [an `object` to be passed](https://github.com/python/typeshed/blob/6d19a17f57762815426942c21d0fadc91aa1e817/stdlib/builtins.pyi#L432-L434).

But if there's a good reason to switch to `Any`, I don't think this is a big deal

Reply:
I see, think we can align and keep `object` then.